src/video.h

```diff
@@ -61,7 +61,9 @@ class VideoSystem {
 	std::string Initialize(const std::string &font_name, int font_size);
 	bool SetResolution(const Point32 &res);
 	void GetResolutions();
-	void MainLoop();
+	static void MainLoop();
+	static bool MainLoopCycle();
```

Comment:
Wouldn't it be useful to keep the static methods as a separate list from the non-static methods?

Otherwise looks fine, please consider and feel free to merge when done.